packages/eclipse/src/styles/globals.css

```
@@ -4,746 +4,364 @@
+  --spacing-2xl: 2.25rem;
+  --spacing-3xl: 2.5rem;
+  --spacing-4xl: 3rem;
+  --spacing-5xl: 4rem;
+  --spacing-6xl: 6rem;
+
+  /* Typography - Font Families */
+  --font-sans-display: "Mona Sans", Inter, Roboto, "Helvetica Neue", "Arial Nova", "Nimbus Sans", Arial, sans-serif;
+  --font-sans-display-settings: 'ss01' on, 'ss02' on, 'ss05' on, 'ss06' on;
+  --font-sans: Inter, Roboto, "Helvetica Neue", "Arial Nova", "Nimbus Sans", Arial, sans-serif;
+  --font-sans-settings: 'cv01' on, 'cv02' on, 'cv06' on, 'cv07' on, 'cv08' on, 'cv10' on;
+  --font-mono: "Mona Sans Mono", ui-monospace, 'Cascadia Code', 'Source Code Pro', Menlo, Consolas, 'DejaVu Sans Mono', monospace;
```

Stylelint (17.3.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)